src/CommunityToolkit.Maui.Core/Essentials/FolderPicker/FolderPicker.shared.cs

```diff
@@ -11,10 +13,20 @@ public static class FolderPicker
 	public static IFolderPicker Default => defaultImplementation.Value;
 
 	/// <inheritdoc cref="IFolderPicker.PickAsync(string, CancellationToken)"/> 
+	[SupportedOSPlatform("Android26.0")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
 	public static Task<FolderPickerResult> PickAsync(string initialPath, CancellationToken cancellationToken) =>
 		Default.PickAsync(initialPath, cancellationToken);
 
-	/// <inheritdoc cref="IFolderPicker.PickAsync(CancellationToken)"/> 
+	/// <inheritdoc cref="IFolderPicker.PickAsync(CancellationToken)"/>
+	[SupportedOSPlatform("Android")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
```

Comment:
I'm not aware of any problems with this feature on iOS 13 and lower? Does this feature not work on iOS 13, for example? 

If there is a known-issue on iOS 13, let's open an Issue and comment here with the link. 

If not, we can remove all of these attributes.

```suggestion
```